onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -829,6 +829,37 @@ class PlannerImpl {
     return p_provider->GetOrtDeviceByMemType(utils::IsInputOnCpu(node, &kernel_create_info, input_index) ? OrtMemTypeCPUInput : OrtMemTypeDefault);
   }
 
+  std::vector<std::pair<int, int>> GetAliasMap(const Node& node, const KernelCreateInfo& kernel_create_info) {
+    if (kernel_create_info.kernel_def != nullptr) {
+#ifdef ENABLE_TRAINING_TORCH_INTEROP
+      if ((node.OpType().compare("PythonOp") == 0 || node.OpType().compare("PythonOpGrad") == 0) &&
+          node.Domain() == kMSDomain) {
+        const auto& attrs = node.GetAttributes();
+        auto attr_it = attrs.find("tensor_reuse_map");
+        if (attr_it != attrs.end()) {
+          const auto& inplace_map = attr_it->second.ints();
```

Comment:
have a few questions regarding this. Let;s discuss offline.

Reply:
Sure no problem.